### PR TITLE
[#149977380] Twitter and Facebook links are broken on Dojo Details page

### PR DESCRIPTION
**Pull request by grabartley**

### CODE REVIEW

File: src/dojos/cd-dojo-details.vue [hunk 1]
```diff
@@ -127,6 +127,20 @@
...
+      buildTwitterLink() {
+        const twitterData = this.dojoDetails.twitter;
+        if (/^[a-zA-Z0-9_@]{1,15}$/.test(twitterData)) {
+          return `https://twitter.com/${twitterData}`;
+        }
+        return twitterData;
+      },
+      buildFacebookLink() {
+        const facebookData = this.dojoDetails.facebook;
+        if (/^[a-zA-Z0-9.]{1,}$/.test(facebookData)) {
```
Comment (Wardormeur) [inline src/dojos/cd-dojo-details.vue]: Don't you need to escape the "." here?

Comment (grabartley) [inline src/dojos/cd-dojo-details.vue]: I don't think so, the regex seems to work when tried in the console with values containing '.'

Comment (Wardormeur) [inline src/dojos/cd-dojo-details.vue]: Could you add a test for it, like you did for the "@" ?

Comment (grabartley) [inline src/dojos/cd-dojo-details.vue]: Will write a test for this :)

**Comment by Wardormeur** [comment]
Seems like dots in [ ] already escape the dot so that it doesnt represent "any character". However it's still necessary for ( )

**Pull request merged**